kornia/contrib/vit.py

```diff
@@ -212,7 +212,7 @@ def __init__(
         self.encoder = TransformerEncoder(hidden_dim, depth, num_heads, dropout_rate, dropout_attn)
 
     @property
-    def encoder_results(self):
+    def encoder_results(self) -> List[Tensor]:
```

Comment:
```suggestion
    def encoder_results(self) -> list[Tensor]:
```

Reply:
I believe that those conversions can be automated via https://beta.ruff.rs/docs/rules/#pyupgrade-up